floss/main.py

```diff
@@ -922,7 +933,20 @@ def print_file_meta_info(vw, selected_functions):
         floss_logger.error("Failed to print vivisect analysis information: {0}".format(e.message))
 
 
-def load_workspace(sample_file_path, save_workspace):
+def populate_function_names(vw, complementary_idb_file_path):
+    with idb.from_file(complementary_idb_file_path) as db:
+        api = idb.IDAPython(db)
+
+        for fva in api.idautils.Functions():
+            function_name = api.idc.GetFunctionName(fva)
+            if vw.isFunction(fva):
+                vw.makeName(fva, function_name)
+                floss_logger.debug('IDA Pro name imported : 0x%x:%s' % (fva, function_name))
+            else:
+                floss_logger.debug('IDA Pro name ignored  : 0x%x:%s' % (fva, function_name))
+
+
+def load_workspace(sample_file_path, complementary_idb_file_path, save_workspace):
```

Comment:
lets make the idb path a kwarg, since its optional.

Reply:
I'm not sure that my Python foo is strong enough here.
Can you give an example on how that would look like ?

Reply:
https://github.com/BenjaminSoelberg/flare-floss/commit/d9d5f46090f312e62a49d58c4abcc4e1c417d8b1